### PR TITLE
Sync UI

### PR DESCRIPTION
**Pull request by harupy**
## Related Issues/PRs

 #xxx

## What changes are proposed in this pull request?

(Please fill in changes proposed in this fix)

## How is this patch tested?

- [x] Existing unit/integration tests
- [ ] New unit/integration tests
- [x] Manual tests (describe details, including test results, below)

## Does this PR change the documentation?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly in the documentation preview.

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by harupy** [comment]
@mlflow-automation autoformat

File: mlflow/server/js/scripts/package-scripts.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```
Comment (xanderwebs) [inline mlflow/server/js/scripts/package-scripts.sh:1]: This file looks like it should have some edge blocking, wdyt?

**Review by sunishsheth2009 — APPROVED** [review]

**Review by xanderwebs — APPROVED** [review]
this is gonna be impossible to review so... lgtm! :shipit:

File: mlflow/server/js/scripts/package-scripts.sh (hunk 1 above)
Comment (harupy) [reply mlflow/server/js/scripts/package-scripts.sh:1]: SGTM. Should we edge-block commands like `databricks-webpack start`?

Comment (xanderwebs) [reply mlflow/server/js/scripts/package-scripts.sh:1]: Yeah cause that doesn't exist here; not the biggest deal though if this is not easy

Comment (harupy) [reply mlflow/server/js/scripts/package-scripts.sh:1]: I think it's fine to include them because we don't touch them on OSS.

**Comment by harupy** [comment]
Interesting. `yarn install` modifies `yarn.lock` on Windows, but doesn't on Linux

https://github.com/mlflow/mlflow/actions/runs/4696257745/jobs/8326189173?pr=8229

**Comment by hubertzub-db** [comment]
> Interesting. yarn install modifies yarn.lock on Windows, but doesn't on Linux

@harupy `YARN_ENABLE_IMMUTABLE_INSTALLS` env might be disabled but using syntax working in shell env only, hence the warning on Windows 🤔. Which change caused it to pass on Windows again?

**Comment by harupy** [comment]
@hubertzub-db https://github.com/mlflow/mlflow/pull/8229/commits/b41292081fe48884f7e610432116e28ee45c70a7 is the change to make CI pass on windows.

**Pull request merged**